thetis/tracer_eq_2d.py

```diff
@@ -0,0 +1,275 @@
+r"""
+2D advection diffusion equation for tracers.
+
+The advection-diffusion equation of tracer :math:`T` in conservative form reads
```

Comment:
is T the depth averaged tracer concentration?

Reply:
It is indeed, so the tracer equation should *not* be solved in conservative form as it suggests in the comment. The implementation indeed uses the non-conservative form, and the docstring for the HorizontalAdvectionTerm does have the right formula.